tests/test_presentation_p4_family_629.py

```
@@ -0,0 +1,567 @@
+            "assert ul.URGE_TRUTH_BANNED_PLAYER_TOKENS is "
+            "dr.URGE_TRUTH_BANNED_PLAYER_TOKENS\n"
+        ),
+        (
+            "import ming_sim.due_review as dr\n"
+            "import ming_sim.urge_lever as ul\n"
+            "assert ul.URGE_TRUTH_BANNED_PLAYER_TOKENS is "
+            "dr.URGE_TRUTH_BANNED_PLAYER_TOKENS\n"
+        ),
+    )
+    for script in scripts:
+        completed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)